locale/de.json

```diff
@@ -390,6 +390,7 @@
     "Mossy Lure Module":"Moos-Lockmodul",
     "Mossy Lure":"Moos Lockmodul",
     "mossy":"moos",
+    "move":"Attacke",
```

Comment:
this is used as a parameter for track, does it work with upper case A?

Reply:
yes

Reply:
yes it works